### PR TITLE
Role fixes

### PR DESCRIPTION
**Pull request by Tunguso4ka**
Fixes supervisors. (When i did roles i set them a little bit wrong)
Fixes notify crew. (Only command roles are now have notify)

### CODE REVIEW

File: Resources/Locale/en-US/_CM14/job/job-supervisors.ftl [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+job-supervisors-cm14-wy = the Wey-Yu corporate office
+job-supervisors-cm14-aso = the auxiliary support officer
+job-supervisors-cm14-co = the acting commanding officer
+job-supervisors-cm14-cmo = the chief medical officer
+job-supervisors-cm14-ares = ARES and APOLLO
+job-supervisors-cm14-po = the pilot officers
+job-supervisors-cm14-uscm = USCM high command
+job-supervisors-cm14-cmp = the Chief MP
+job-supervisors-cm14-qm = the quartermaster
+job-supervisors-cm14-ce = the chief engineer
```
Comment (QuietlyWhisper) [inline Resources/Locale/en-US/_CM14/job/job-supervisors.ftl:11]:
```suggestion
job-supervisors-cm14-ce = the Chief Engineer
```

File: Resources/Locale/en-US/_CM14/job/job-supervisors.ftl [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-job-supervisors-warden = the Warden
-job-supervisors-chiefmp = the Chief Military Police
-job-supervisors-cm-quartermaster = the Quartermaster
-job-supervisors-cm-chief-engineer = the Chief Engineer
-job-supervisors-high-command = USCM High Command
-job-supervisors-cm-cmo = the Chief Medical Officer
-job-supervisors-nobody = nobody, you are on your own.
+job-supervisors-cm14-wy = the Wey-Yu corporate office
+job-supervisors-cm14-aso = the auxiliary support officer
+job-supervisors-cm14-co = the acting commanding officer
```
Comment (QuietlyWhisper) [inline Resources/Locale/en-US/_CM14/job/job-supervisors.ftl:4]:
```suggestion
job-supervisors-cm14-co = the Commanding Officer
```

File: Resources/Locale/en-US/_CM14/job/job-supervisors.ftl [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
-job-supervisors-squad-leader = the Squad Leader
-job-supervisors-warden = the Warden
-job-supervisors-chiefmp = the Chief Military Police
-job-supervisors-cm-quartermaster = the Quartermaster
-job-supervisors-cm-chief-engineer = the Chief Engineer
-job-supervisors-high-command = USCM High Command
-job-supervisors-cm-cmo = the Chief Medical Officer
-job-supervisors-nobody = nobody, you are on your own.
+job-supervisors-cm14-wy = the Wey-Yu corporate office
+job-supervisors-cm14-aso = the auxiliary support officer
```
Comment (QuietlyWhisper) [inline Resources/Locale/en-US/_CM14/job/job-supervisors.ftl:3]:
```suggestion
job-supervisors-cm14-aso = the Auxiliary Support Officer
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/_CM14/job/job-supervisors.ftl (hunk 1 above)
Comment (Tunguso4ka) [reply Resources/Locale/en-US/_CM14/job/job-supervisors.ftl]: done.

File: Resources/Locale/en-US/_CM14/job/job-supervisors.ftl (hunk 2 above)
Comment (Tunguso4ka) [reply Resources/Locale/en-US/_CM14/job/job-supervisors.ftl]: done.

File: Resources/Locale/en-US/_CM14/job/job-supervisors.ftl (hunk 3 above)
Comment (Tunguso4ka) [reply Resources/Locale/en-US/_CM14/job/job-supervisors.ftl]: done.

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**